libs/grout/src/client.ts

```diff
@@ -1,12 +1,20 @@
 /* eslint-disable vx/gts-identifiers */
 /* eslint-disable vx/gts-type-parameters */
+import nodeFetch from 'node-fetch';
```

Comment:
Is this elided on the client, or are we pulling in all of `node-fetch` in the browser?

Reply:
I think the way I have it now, it will be imported in the browser. Do you have any ideas on how to leave it out?

Reply:
I think we'd have to stub it in any of the frontends that pull in `grout`. Or maybe it's not actually an issue and we should try building with this configuration and see if `node-fetch` gets pulled in.

Reply:
Is the goal just to avoid pulling in unnecessary code? Or do you think maybe the build won't work due to node-only dependencies? Or something else?

Reply:
Both.

Reply:
Finally tested this out. node-fetch did cause issues in the browser (it assumes on node's Readable is in scope), so I switched to [cross-fetch](https://github.com/lquixada/cross-fetch), which is designed for this purpose, and that worked.